frontend/src/components/accountMgmt/ManageUsers.vue

```diff
@@ -9,18 +57,81 @@
   </v-container>
 </template>
 <script>
+import { mapState, mapActions } from 'pinia';
+import { isEmpty } from 'lodash';
 import { PATHS } from '@/utils/constants.js';
+import contactService from '@/services/contactService.js';
+import { useOrganizationStore } from '@/store/ccof/organization';
 
+import alertMixin from '@/mixins/alertMixin.js';
+import AppButton from '@/components/guiComponents/AppButton.vue';
 import NavButton from '@/components/util/NavButton.vue';
 
 export default {
-  name: 'ManageUser',
-  components: { NavButton },
+  name: 'ManageUsers',
+  components: { AppButton, NavButton },
+  mixins: [alertMixin],
   data() {
     return {
       tab: undefined,
       PATHS,
+      contacts: [],
+      sortBy: [{ key: 'firstName', order: 'asc' }],
+      contactsLoading: false,
     };
   },
+  computed: {
+    ...mapState(useOrganizationStore, [
+      'organizationId',
+      'organizationName',
+      'organizationAccountNumber',
+      'loadedModel',
+    ]),
+    headers() {
+      return [
+        { title: '', key: 'edit-user', sortable: false },
+        { title: 'First Name', key: 'firstName' },
+        { title: 'Last Name', key: 'lastName' },
+        { title: 'Phone Number', key: 'telephone' },
+        { title: 'Email', key: 'email' },
+        {
+          title: 'Access Type',
+          key: 'access-type',
+          sortable: false,
+        },
+        { title: '', key: 'remove-user', align: 'end', sortable: false },
+      ];
+    },
+  },
+  async mounted() {
+    try {
+      this.contactsLoading = true;
+      if (isEmpty(this.loadedModel)) {
+        await this.loadOrganization(this.organizationId);
+      }
+      this.contacts = await contactService.loadContacts(this.organizationId);
+    } catch (error) {
+      this.setFailureAlert('There was an error loading the users');
+      console.error('Error loading users: ', error);
+    } finally {
+      this.contactsLoading = false;
+    }
+  },
+  methods: {
+    ...mapActions(useOrganizationStore, ['loadOrganization']),
+    isContactOnly(contact) {
```

Comment:
For the future, if we find that isContactOnly (or even accessType) is a useful property in multiple screens it might be useful to set it directly in the backend through custom code after the Mapping is applied.